### PR TITLE
add sig-scheduling multizone job with kind

### PR DESCRIPTION
**Pull request by aojea**

### CODE REVIEW

File: config/jobs/kubernetes/sig-scheduling/sig-scheduling-config.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        privileged: true
+      resources:
+        requests:
+          memory: "9000Mi"
+          cpu: 4000m
+  annotations:
+    testgrid-dashboards: sig-scheduling-kind
+    testgrid-tab-name: sig-scheduling-kind, multizone
+    description: Runs tests against a Multizone Kubernetes in Docker cluster
+    testgrid-alert-email: antonio.ojea.garcia@gmail.com, alculquicondor@google.com
```
Comment (aojea) [inline config/jobs/kubernetes/sig-scheduling/sig-scheduling-config.yaml:45]: @alculquicondor I made up the email :smile: , please let me know which addresses do you want here?

Comment (alculquicondor) [reply config/jobs/kubernetes/sig-scheduling/sig-scheduling-config.yaml:45]: `acondor@google.com` please. LGTM otherwise

Comment (aojea) [inline config/jobs/kubernetes/sig-scheduling/sig-scheduling-config.yaml:45]:
```suggestion
    testgrid-alert-email: antonio.ojea.garcia@gmail.com, acondor@google.com
```

File: config/testgrids/kubernetes/sig-network/config.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ dashboard_groups:
     - sig-network-gateway-api
     - sig-network-ingress-controller-conformance
     - sig-network-kind
+    - sig-scheduling-kind
```
[the remark this answers is not in the document]
Comment (aojea) [reply config/testgrids/kubernetes/sig-network/config.yaml:13]: lol, copy paste

**Comment by aojea** [comment]
>     config_test.go:280: Duplicated testgroupnames in dashboard sig-scheduling: ci-kubernetes-kind-scheduling-multizone-serial

I always have troubles with this :(

**Comment by aojea** [comment]
lol, it only needed to match the tab name
finally @alculquicondor , green

**Comment by alculquicondor** [comment]
Thanks!
/lgtm

**Comment by aojea** [comment]
cool, it is working https://testgrid.k8s.io/sig-scheduling#sig-scheduling-kind,%20multizone&include-filter-by-regex=Multi-AZ

**Comment by alculquicondor** [comment]
FYI @msau42, in case you want to "unskip" the volume test. Although I suspect that actually requires a cloud provider?

**Comment by msau42** [comment]
> FYI @msau42, in case you want to "unskip" the volume test. Although I suspect that actually requires a cloud provider?

Yeah the volume tests require a cloud provider, which we should cover as part of cloud-provider-gcp tests cc @mattcary

**Comment by mattcary** [comment]
> > FYI @msau42, in case you want to "unskip" the volume test. Although I suspect that actually requires a cloud provider?
> 
> Yeah the volume tests require a cloud provider, which we should cover as part of cloud-provider-gcp tests cc @mattcary

Yes, although there don't seem to be any gce-pd tests run? [testgrid](https://testgrid.k8s.io/provider-gcp-presubmits#cloud-provider-gcp-e2e-full&include-filter-by-regex=gce-pd) 

Although maybe I'm looking at the wrong testgrid @jpbetz

**Comment by mattcary** [comment]
> Multizone volume tests are here: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/ubernetes_lite_volumes.go

Weird, it shows up as passing with skips in the overview, but when I look at the logs I can't find it. https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/cloud-provider-gcp/312/cloud-provider-gcp-e2e-full/1479250385913778176/build-log.txt

**Pull request merged**